src/gluonts/dataset/split/splitter.py

```diff
@@ -268,4 +268,9 @@ def _train_slice(self, item: TimeSeriesSlice) -> TimeSeriesSlice:
     def _test_slice(
         self, item: TimeSeriesSlice, offset: int = 0
     ) -> TimeSeriesSlice:
-        return item[: self.split_date + self.prediction_length + offset]
+        freq = item.start.freq
+        return item[
+            : self.split_date
+            + pd.Timedelta(self.prediction_length, unit=freq)
+            + pd.Timedelta(offset, unit=freq)
```

Comment:
```suggestion
            + pd.Timedelta(self.prediction_length + offset, unit=freq)
```

I think we can simplify this a bit.